### PR TITLE
Kill process if term fails, add status sub-command

### PR DESCRIPTION
**Pull request by Garreth-BetBright**

### CODE REVIEW

**Comment by MarchettaPietro** [comment]
External review passed :+1:

File: init.sh [hunk 1]
```diff
@@ -102,22 +105,31 @@ do_stop() {
         echo Not running
         return 0
     fi
-    echo Stopping...
-    kill $(cat $PIDFILE) || return 1
+    echo -n "Stopping... "
+    kill $(cat $PIDFILE)
```
Comment (Stqs) [inline init.sh]: signal is not specified

Comment (rhojland) [inline init.sh]: Signal number for TERM is not need to and excessively verbose.
There’s some other serious underlying issues to the whole way this works but it’ll have to do for now.

Ragnar

**Comment by MarchettaPietro** [comment]
External review passed :+1:

**Pull request merged**